index.d.ts

```diff
@@ -32,4 +32,23 @@ declare module "effection" {
   export function join(context: Context): Operation;
 
   export function timeout(durationMillis: number): Operation;
+
+
+  // send/receive operations, will probably be moved to library at some point.
```

Comment:
I put these at the bottom of the type declarations since I think it's likely that we'll extract this into an Addon package like `@effection/mailbox` or `@effection/events`